src/V3Width.cpp

```diff
@@ -3672,6 +3651,66 @@ class WidthVisitor final : public VNVisitor {
         }
         VL_DO_DANGLING(pushDeletep(nodep), nodep);  // Deletes defaultp also, if present
     }
+
+    AstNode*
+    nestedvalueConcat_patternUOrStruct(AstNodeUOrStructDType* memp_vdtypep, AstPatMember* defaultp,
+                                       AstNode* newp, AstPattern* nodep,
+                                       std::map<const std::string, AstPatMember*> dtypemap) {
```

Comment:
Add a "using" typedef at the top of the class for this std::map, since use it multiple places.